specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/registries.json

```diff
@@ -0,0 +1,911 @@
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "PartialRegistry": {
+      "description": "Partial Registry class for PATCH",
+      "type": "object",
+      "additionalProperties": false
+    },
+    "PartialRegistryPartialTrackedResource": {
+      "description": "Strictly used in update requests.",
+      "type": "object",
+      "properties": {
+        "identity": {
+          "description": "Managed service identity (system assigned and/or user assigned identities)",
+          "$ref": "#/definitions/PartialManagedServiceIdentity"
+        },
+        "kind": {
+          "description": "Metadata used by portal/tooling/etc to render different UX experiences for resources of the same type.",
+          "type": "string"
+        },
+        "location": {
+          "description": "The geo-location where the resource lives.",
+          "type": "string"
+        },
+        "properties": {
+          "description": "Additional attributes of the entity.",
+          "$ref": "#/definitions/PartialRegistry"
+        },
+        "sku": {
+          "description": "Sku details required for ARM contract for Autoscaling.",
+          "$ref": "#/definitions/PartialSku"
+        },
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "x-nullable": true
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "PartialSku": {
+      "description": "Common SKU definition.",
+      "type": "object",
+      "properties": {
+        "capacity": {
+          "format": "int32",
+          "description": "If the SKU supports scale out/in then the capacity integer should be included. If scale out/in is not possible for the resource this may be omitted.",
+          "type": "integer"
+        },
+        "family": {
+          "description": "If the service has different generations of hardware, for the same SKU, then that can be captured here.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the SKU. Ex - P3. It is typically a letter+number code.",
+          "type": "string"
+        },
+        "size": {
+          "description": "The SKU size. When the name field is the combination of tier and some other value, this would be the standalone code.",
+          "type": "string"
+        },
+        "tier": {
+          "description": "This field is required to be implemented by the Resource Provider if the service has more than one tier, but is not required on a PUT.",
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/SkuTier"
+        }
+      },
+      "additionalProperties": false
+    },
+    "PartialUserAssignedIdentity": {
+      "type": "object",
+      "additionalProperties": false
+    },
+    "PrivateEndpointConnectionARMResource": {
```

Comment:
>PrivateEndpointConnectionARMResource [](http://example.com/codeflow?start=5&length=36)

use from common types : [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v4/privatelinks.json#L111](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v4/privatelinks.json#L111)

Reply:
PrivateLinkResource has its properties defined differently from what we want them to be (i.e. PrivateEndpointConnectionProperties), so can't use this either